src/chimera/compositions.cljc

```diff
@@ -0,0 +1,4 @@
+(ns chimera.compositions)
```

Comment:
I feel like `core` would be a better name for this, since these are `core` functions, like if we have a composition that's related to strings it would be easier to find in a `string` namespace, not here, you know?